src/kemal/ext/context.cr

```diff
@@ -17,8 +17,9 @@ class HTTP::Server
       @params ||= Kemal::ParamParser.new(@request, route_lookup.params)
     end
 
-    def redirect(url : String, status_code : Int32 = 302)
+    def redirect(url : String, status_code : Int32 = 302, *, response = "")
```

Comment:
The argument should be called `body`. And maybe `nil` would be a better default, no need to allocate a string.

Reply:
:ok_hand: